apps/communications/contacts/views/form/js/boot.js

```diff
@@ -43,8 +43,47 @@ window.addEventListener('load', function() {
   LazyLoader.load(
     dependencies,
     function() {
-      FormUI.init();
+
+      window.addEventListener('renderdone', function fn() {
+        window.removeEventListener('renderdone', fn);
+        document.body.classList.remove('hidden');
+      });
+
+      function getParams() {
+        var params = {};
+        var raw = window.location.search.split('?')[1];
+        var pairs = raw.split('&');
+        for (var i = 0; i < pairs.length; i++) {
+
+          var data = pairs[i].split('=');
+          params[data[0]] = data[1];
+        }
+        return params;
+      }
```

Comment:
Maybe we could extract this function to a separate .js, in order to load it when necessary (In almost all activities). It's just a thought, to avoid copy&paste.